php/storage/framework/cache/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
Do we need these? Will the base gitignore not cover it?

Reply:
They ship with the default PHP Laravel Lumen skeleton project:
- https://github.com/laravel/lumen/tree/8.x/storage

Removing the folders leads to an exception that the cache directory is broken:
- https://github.com/Paymentsense/connect-e-samples/pull/5#discussion_r711031095

I could swap the `.gitignore` files for one parent file, but I'd need to add `.gitkeep` files as git does not allow empty folders to be committed. It's easier to just leave the default structure with the default files. It's messy I know, but so is PHP in general 😁